frontend/src/pages/LoginPage/LoginPage.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/no-static-element-interactions */
 /* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
Seams that you use wrong tags again, every elemente with onClick prop need to be accessible by keyboard events. I think that if you correct the issues above this is not necessary. 

About no-shadow, indicates that in your code you are using same name of variables in scope, this can lead a confuse code, trying to review your code and always use distinc names for variables even in diff scopes.